modules/microshift-custom-ca-con.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-custom-cas_{context}"]
+= How custom certificate authorities work in {microshift-short}
+
+The default API server certificate is issued by an internal {microshift-short} cluster certificate authority (CA). Clients outside of the cluster cannot verify the API server certificate by default. This certificate can be replaced by a custom server certificate that is issued externally by a custom CA that clients trust. The following steps illustrate the workflow in {microshift-short}:
+
+. Set up your CAs. Updating the certificate bundle with the configuration that contains the CA is recommended. If you do not want to configure your certificate bundles, you can alternately use the `oc login localhost:8443 --certificate-authority=/path/to/cert.crt` command, but this method is more manual and not preferred.
+. Specify the CA names in the {microshift-short} configuration file, including the new fully qualified domain name (FQDN).
+** In most cases, {microshift-short} generates a new `kubeconfig` for your custom CA. The exception is when wildcards are specified for the IP address. In this case, {microshift-short} generates a `kubeconfig` with the public IP address of the server. You can update that generic file with your specific details.
+** You can also provide additional DNS names for each certificate.
+. After system restart, the {microshift-short} service generates the `kubeconfig` files of your custom certificates.
+. If needed, you can further configure your certificates and domain names by editing the {microshift-short}-generated `kubeconfig`. Each certificate can contain the following values:
```

Comment:
the certificate domain names configuration done in `/etc/microshift/config.yaml` not in generated `kubeconfig`